cmd/ui/src/components/LoginViaSSOForm/LoginViaSSOForm.tsx

```diff
@@ -18,13 +18,13 @@ import { Button } from '@bloodhoundenterprise/doodleui';
 import { Box, FormControl, Grid, InputLabel, MenuItem, Select } from '@mui/material';
 import React from 'react';
 
-interface LoginViaSAMLFormProps {
-    providers: { name: string; initiation_url: string }[];
+interface LoginViaSSOFormProps {
+    providers: { name: string; slug: string }[] | undefined;
```

Comment:
I can put that in.

I was debating on this since the LoginViaSSOForm only requires these two fields to be set rather than a whole SSOProvider, it felt easier to understand what needs to be passed in rather than constructing a new object to satisfy the requirement.